Backend/src/index.ts

```diff
@@ -0,0 +1,66 @@
+import express from 'express';
+import dotenv from 'dotenv';
+
+import { connectToDatabase } from './databaseConnection';
+import { userRoute } from './routes/user.routes';
+import { authRoute } from './routes/auth.routes';
+import { clusterRoute } from './routes/cluster.routes';
+import { graphRoute } from './routes/graph.routes';
+import authMiddleware from './middleware/auth.middleware';
+import { exec } from 'child_process';
+import fs from 'fs';
+import path from 'path';
+
+dotenv.config();
+
+const HOST = process.env.HOST || 'http://localhost';
+const PORT = parseInt(process.env.PORT || '8080');
+
+console.log('MONGO:', process.env.MONGO_URL);
+
+const app = express();
+
+app.use('/public', express.static('public'));
+
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
+
+app.use('/auth', authRoute());
+app.use('/users', userRoute());
+app.use('/clusters', authMiddleware,  clusterRoute());
+app.use('/graph', graphRoute());
+
+app.get('/', (req, res) => {
+  return res.json({ message: 'Hello World!' });
```

Comment:
Why do we return a "Hello World!" message here?

Reply:
this one is used for testing, removed it